lib/api-client.js

```diff
@@ -152,22 +155,22 @@ const getById = async (apiPath, id, accessToken, apiVersion, queryParam) => {
   }
 }
 
-const getInvitationById = async (invitationId, accessToken, queryParam1, queryParam2) => {
-  const invitationObj = await getById('invitations', invitationId, accessToken, 2, queryParam1)
+const getInvitationById = async (invitationId, accessToken, queryParam1, queryParam2, remoteIp) => {
+  const invitationObj = await getById('invitations', invitationId, accessToken, 2, queryParam1, remoteIp)
   if (invitationObj) {
     return invitationObj
   }
 
-  return getById('invitations', invitationId, accessToken, 1, queryParam2 ?? queryParam1)
+  return getById('invitations', invitationId, accessToken, 1, queryParam1 ?? queryParam2, remoteIp)
```

Comment:
i think queryParam1 here is the v2 param
so it should check queryParam2 (v1 param) first

arr invitation edit page can be used for testing

Reply:
I see what you mean. I changed this line to match the behavior of getNoteById, but I guess that was the one that had the bug.

Reply:
i see
please take a look at #1794